environments/dev/terraform/clinvar.tf

```diff
@@ -19,5 +19,5 @@ module clinvar {
   k8s_scaled_machine_type = "n1-standard-1"
   k8s_static_cluster_size = 2
   reader_groups = ["clingendevs@broadinstitute.org"]
-  deletion_age_days = 30
+  deletion_age_days = null # FIXME: Reset back to 30 after we set up prod.
```

Comment:
ClinGen are experimenting with the files in this bucket, we don't want to delete anything out from under them.

Reply:
Ticket already created to switch this back?

Reply:
DSPDC-731